ui/src/components/ApprovalsTracker/index.tsx

```diff
@@ -356,7 +356,7 @@ const ApprovalsTracker: React.FC<Props> = (props): React.ReactElement => {
                 <Components.Modal
                     open={Boolean(selectedAuthorEmail)}
                     icon={<FaIcons.FaEdit className="h-8 w-8 text-grey-600" />}
-                    setOpen={handleCloseChangeEmailModal}
+                    onClose={handleCloseChangeEmailModal}
```

Comment:
![image](https://github.com/JiscSD/octopus/assets/48569671/de679023-fb6c-4202-b8ad-afa1b49fa4ca)

The modal size changed and the buttons look bad because of that

Reply:
This seems to be fixed by re-adding as={React.Fragment} to the transition child on the modal. The loading bar still seems to work fine with this change